apps/explorer/src/ui/TableCard.tsx

```diff
@@ -56,80 +43,34 @@ type TxType = {
         | TxStatus;
 };
 
-export function TxAddresses({ content }: { content: LinkObj[] }) {
-    return (
-        <section className={styles.addresses}>
-            {content.map((itm, idx) => (
-                <Fragment key={idx + itm.url}>
-                    <Longtext
-                        text={itm.url}
-                        alttext={itm.name}
-                        category={(itm.category as Category) || 'unknown'}
-                        isLink={itm?.isLink}
-                        copyButton={itm?.copy ? '16' : 'none'}
-                    />
-                    {idx !== content.length - 1 && <ContentArrowRight />}
-                </Fragment>
-            ))}
-        </section>
-    );
-}
-
-function TxStatusType({ content }: { content: TxStatus }) {
-    const TxStatus = {
-        success: ContentSuccessStatus,
-        fail: ContentFailedStatus,
-    };
-    const TxResultStatus =
-        content.status === 'success' ? TxStatus.success : TxStatus.fail;
-    return (
-        <section className={styles.statuswrapper}>
-            <div
-                className={
-                    content.status === 'success' ? styles.success : styles.fail
-                }
-            >
-                <TxResultStatus /> <div>{content.txTypeName}</div>
-            </div>
-        </section>
-    );
-}
-
 function columnsContent(columns: TableColumn[]) {
     return columns.map((column) => ({
         accessorKey: column.accessorKey,
         id: column.accessorKey,
         header: column.headerLabel,
         // cell renderer for each column from react-table
-        cell: (info: any) => {
-            const content = info.getValue();
-
-            // handle multiple links in one cell
-            if (Array.isArray(content)) {
-                return <TxAddresses content={content} />;
-            }
-            // Special case for txTypes and status
-            if (
-                typeof content === 'object' &&
-                content !== null &&
-                content.txTypeName
-            ) {
-                return <TxStatusType content={content} />;
-            }
-            // handle most common types
-            return <section>{content}</section>;
-        },
+        cell: (info: any) => info.getValue(),
     }));
 }
 
-function TableCard({
-    tabledata,
-}: {
-    tabledata: {
-        data: TxType[];
-        columns: TableColumn[];
-    };
-}) {
+export type TableType = {
+    data: DataType[];
+    columns: TableColumn[];
+};
+
+const cellStyle = cva(['text-sui-grey-75 h-[30px] px-[4px]'], {
+    variants: {
+        variant: {
+            th: 'text-left font-semibold uppercase text-xs',
+            td: 'group-hover:text-sui-grey-90 group-hover:bg-sui-grey-40 text-sm group-hover:first:rounded-l-[4px] group-hover:last:rounded-r-[4px]',
+        },
+    },
+    defaultVariants: {
+        variant: 'td',
+    },
+});
+
+export function TableCard({ tabledata }: { tabledata: TableType }) {
```

Comment:
Given this is a UI component you should follow the convention here of creating an exported props interface named `TableCardProps`.